schemas/context/device.schema.json

```diff
@@ -44,7 +44,9 @@
             "gaming": "Gaming Console",
             "television": "Television",
             "settop": "Set-top Box",
-            "mediaplayer": "Media Player"
+            "mediaplayer": "Media Player",
+            "computers": "Computers",
+            "tv screens": "TV Screens"
```

Comment:
@khanna-vikas - Note this field has a meta:enum with no enum so technically it has not constrained values. Also note that value on "tv screens" has a space. This may be okay, but just something to be aware of.

Reply:
meta:enums are fine, unless they are spelled wrongly.